pkg/operators/v1/operatorgroup_types.go

```diff
@@ -45,9 +45,32 @@ type OperatorGroupSpec struct {
 	// Static tells OLM not to update the OperatorGroup's providedAPIs annotation
 	// +optional
 	StaticProvidedAPIs bool `json:"staticProvidedAPIs,omitempty"`
+
+	// UpgradeStrategy defines the upgrade strategy for operators in the namespace.
+	// There are currently two supported upgrade strategies:
+	//
+	// Default: OLM will only allow clusterServiceVersions to move to the replacing
+	// phase from the succeeded phase. This effectively means that OLM will not
+	// allow operators to move to the next version if an installation or upgrade
+	// has failed.
+	//
+	// TechPreviewUnsafeFailForward: OLM will allow clusterServiceVersions to move to the
+	// replacing phase from the succeeded phase or from the failed phase.
+	// Additionally, OLM will generate new installPlans when a subscription references
+	// a failed installPlan and the catalog has been updated with a new upgrade for
+	// the existing set of operators.
+	//
+	// WARNING: The TechPreviewUnsafeFailForward upgrade strategy is unsafe and may result
+	// in unexpected behavior or unrecoverable data loss unless you have deep
+	// understanding of the set of operators being managed in the namespace.
+	// OperatorGroupStatus is the status for an OperatorGroupResource.
```

Comment:
Looks like this got lot in the most recent changes:

```go
	// OperatorGroupStatus is the status for an OperatorGroupResource.
```

Is in the UpgradeStrategy godoc vs. above the OperatorGroupStatus type.

Reply:
Oops, will fix